docs/docs/who-is-using-msys2.md

```diff
@@ -0,0 +1,7 @@
+# Who Is Using MSYS2?
+
+* [Git for Windows](https://gitforwindows.org/) is based on MSYS2
+* [Quod Libet](https://quodlibet.readthedocs.io) uses MSYS2 for its Windows
+  build and for CI
+
+*Feel free to add your project to this list!*
```

Comment:
I assembled some other users in Devtopics.

Reply:
Thanks, I'll move them over later